src/api.js

```diff
@@ -0,0 +1,19 @@
+const BASE_URL = "https://bootcamp-api.codeit.kr/api/sample";
+
+export async function getUserInfo() {
+    const response = await fetch(`${BASE_URL}/user`);
+    if (!response.ok) {
+        throw new Error("유저정보를 불러오는데 실패했습니다.");
+    }
+    const body = await response.json();
+    return body;
```

Comment:
제 개인 취향으로는 요렇게 개행이 들어가는 것이긴 한데 지금도 괜찮은 것 같아요~
```suggestion
    const response = await fetch(`${BASE_URL}/user`);
    
    if (!response.ok) {
        throw new Error("유저정보를 불러오는데 실패했습니다.");
    }
    
    const body = await response.json();
    
    return body;
```